README.md

```diff
@@ -159,6 +159,46 @@ func TestFoo(t *testing.T) {
 }
 ```
 
+Table Driven Tests
+------------------
+Ref: [Issue 223](https://github.com/golang/mock/issues/223)  
+It is important to note, especially when testing multiple cases in a loop, to setup your mock controller within your sub-tests.  
+Otherwise, test may yeild unexpected behavior.
```

Comment:
I'm not sure this blurb adds enough. I would just say the following is an example. Maybe add a comment in the code where the controller is initialized that points out it has to be created with the new `t`.